packages/web-components/README.md

```diff
@@ -1,31 +1,33 @@
 # Fluent UI Web Components
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
-[![npm version](https://badge.fury.io/js/%40fluentui%2Fweb-components.svg)](https://badge.fury.io/js/%40fluentui%2Fweb-components)
+[![npm version](https://img.shields.io/npm/v/@fluentui/web-components/beta?style=flat-square)](https://www.npmjs.com/package/@fluentui/web-components/v/beta)
 
-`@fluentui/web-components` is a library of Web Components that _composes_ `@microsoft/fast-foundation` and supports Microsoft's Fluent design language.
+Microsoft's [Fluent UI Web Components](https://github.com/microsoft/fluentui/tree/master/packages/web-components) is designed to help you build web apps using Web Components styled with the [Fluent design language](https://github.com/microsoft/fluentui).
 
 ## Installation
 
-### From NPM
+## Install
 
-To install the `web-components` library, use either `npm` or `yarn` as follows:
+Fluent UI should be installed as a `dependency` of your app.
 
-```shell
-npm install --save @fluentui/web-components
-```
+**Yarn**
 
-```shell
-yarn add @fluentui/web-components
+```sh
+yarn add @fluentui/web-components@beta
 ```
 
-Within your JavaScript or TypeScript code, you can then import library APIs like this:
+**NPM**
 
-```ts
-import { FluentAnchor } from '@fluentui/web-components';
+```sh
+npm i @fluentui/web-components@beta
```

Comment:
```suggestion
npm i --save @fluentui/web-components@beta
```

Reply:
Do we need this in modern versions of NPM (greater than 5.0.0)?